p2p/src/net/mock/peer.rs

```diff
@@ -97,10 +103,9 @@ impl Peer {
                             peer_id: self.local_peer_id,
                             version: *self.config.version(),
                             network: *self.config.magic_bytes(),
-                            protocols: vec![
-                                types::Protocol::new("floodsub", *self.config.version()),
-                                types::Protocol::new("ping", *self.config.version()),
-                            ],
+                            protocols: parse_protocols(["floodsub", "/ipfs/ping/1.0.0"])
```

Comment:
I think this is incorrect. The ping protocol used by the mock interface is not called `/ipfs/ping/1.0.0` because the mock interface has no relation to IPFS. If you want to use a libp2p-like protocol name and not just `ping` as it was before, `/mintlayer/mock/ping/0.1.0` would be an appropriate name.

Reply:
I have reverted the ping protocol name, but with the current `parse_protocols` implementation it produces an empty set. Are we OK with that?

Reply:
yes it looks good